### PR TITLE
Add empty commit option and custom commit message

### PR DESCRIPTION
**Pull request by xxxtonixxx**
Added `empty commit` parameter to allow generate a commit without add files, the command `git add --force` adds every files, gitignore files included!. This only generates a commit with nothing.

And `commit_message` parameter added to allow customize the commit message (by default it's  `[skip ci] Commit dirty state`

### CODE REVIEW

File: repo/commit.go [hunk 1]
```diff
@@ -15,12 +18,35 @@ func ForceAdd() *exec.Cmd {
 	return cmd
 }
 
+// EmptyCommit simply create an empty commit
+func EmptyCommit(msg string) *exec.Cmd {
+	commitMessage := defaultCommitMessage
```
Comment (tboerger) [inline repo/commit.go]: What about just switching it?

```
if msg == "" {
    msg = defaultCommitMessage
}
```

File: repo/commit.go [hunk 2]
```diff
@@ -15,12 +18,35 @@ func ForceAdd() *exec.Cmd {
...
+		"--allow-empty",
+		fmt.Sprintf("-m '%s'", commitMessage))
+
+	return cmd
+}
+
 // ForceCommit commits every change while skipping CI.
-func ForceCommit() *exec.Cmd {
+func ForceCommit(msg string) *exec.Cmd {
+	commitMessage := defaultCommitMessage
```
Comment (tboerger) [inline repo/commit.go]: Same as above

Comment (xxxtonixxx) [inline repo/commit.go]: Ok, it's fixed!

**Comment by tboerger** [comment]
WTF? `cannot find package "github.com/drone-plugins/drone-git-push/repo" in any of`

**Comment by xxxtonixxx** [comment]
I was wondering the same

**Comment by tboerger** [comment]
@bradrydzewski i think this is related to the changed default path, right?

**Comment by appleboy** [comment]
@tboerger @xxxtonixxx Already fix. I will transfer this repo to my account and maintain this repo in the future.

**Comment by appleboy** [comment]
@xxxtonixxx build fail http://beta.drone.io/drone-plugins/drone-git-push/73

**Comment by appleboy** [comment]
LGTM

**Comment by xxxtonixxx** [comment]
Thanks @appleboy, it's fixed 😄

**Pull request merged**